schema/README.md

```diff
@@ -0,0 +1,33 @@
+# JSON schema
+
+## Overview
+
+This directory contains the [JSON Schema](http://json-schema.org/) for
+validating the `config.json` of this container runtime specification.
+
+The layout of the files is as follows:
+* schema.json - the primary entrypoint for the whole schema document
+* schema-linux.json - this schema is for the Linux specific sub structure
+* defs.json - definitions for general types
+* defs-linux.json - definitions for Linux specific types
+* validate.go - validation utility source code
+ 
+
+## Utility
+
+There is also included a simple utility for facilitating validation of a
+`config.json`. To build it:
+
+```bash
+export GOPATH=`mktemp -d`
+go get ./...
+go build .
+rm -rf $GOPATH
+```
+
+Then use it like:
+
+```bash
+./schema ./schema.json <yourpath>/config.json
```

Comment:
On Wed, Mar 30, 2016 at 12:35:57PM -0700, Vincent Batts wrote:
> > +./schema ./schema.json <yourpath>/config.json
> 
> `./` is common path ...

I agree it's legal; I just don't think it's intuitive, and it strikes
me as an unlikely requirement.  I see things like ‘git add README’ [1]
a lot more frequently than ‘git add ./README’.

[1]: https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#Tracking-New-Files